### DISCUSSION TITLE
posts/tutos/my-first-app/8-media-control/

### DISCUSSION THREAD

**Discussion by giscus** [Kapusch/blog-dotnet-maui#10] [Blog comments]
# posts/tutos/my-first-app/8-media-control/

Des boutons, toujours plus de boutons ! Dans ce chapitre, on va intégrer les contrôles pour manipuler le média.

https://www.mauicestclair.fr/posts/tutos/my-first-app/8-media-control/

**Comment by Mkkaba** [comment]
Cool, merci beaucoup 😊
